src/Google/Client.php

```diff
@@ -753,7 +763,7 @@ public function getScopes()
   }
 
   /**
-   * @return array
+   * @return string
```

Comment:
good catch, should be `string|null` though

Reply:
You are right. I fixed that.